### PR TITLE
NIP-06: Generating keys from mnemonic codes

### PR DESCRIPTION
**Pull request by fiatjaf**

### CODE REVIEW

**Comment by fiatjaf** [comment]
Following discussion on https://t.me/nostr_protocol/6905, we'll close this and use normal BIP39/BIP32 along with SLIP44.

**Comment by fiatjaf** [comment]
https://github.com/satoshilabs/slips/pull/1226

**Comment by fiatjaf** [comment]
OK, NIP-06 is now just a dumb derivation path so we can all be on the same page.

File: nips/06.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+======
+
+Basic key derivation from mnemonic seed phrase
+----------------------------------------------
+
+`draft` `optional` `author:fiatjaf`
+
+[BIP39](https://bips.xyz/39) is used to generate mnemonic seed words and derive a binary seed from them.
+
+[BIP32](https://bips.xyz/32) is used to derive the path `m/44'/1237'/ 0'/ 0'` (according to the Nostr entry on [SLIP44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md)).
```
Comment (Kukks) [inline nips/06.md:11]: I think this should be m/44'/1237'/0'/0/0

Comment (fiatjaf) [reply nips/06.md:11]: Suggestion denied.

Comment (fiatjaf) [reply nips/06.md]: Ok, I screwed up. My implementation was broken, therefore everybody will lose their mnemonics anyway. So I'll change the spec to this.

**Pull request merged**